doc/release.rst

```diff
@@ -0,0 +1,113 @@
+.. _release_notes:
+
+********************************************************************************
+Release notes
+********************************************************************************
+
+Tarantool versions later than 1.10.0 and 2.0.0 follow the same rules:
```

Comment:
1. If this is the first and main page dedicated to releases, maybe it makes sense to say a few extra introductory sentences as we do at the Tarantool Release Policy page that became now a child page. Otherwise, seeing digits like 1.10.0 right at the begging of the explanation can be confusing.
2. And a suggestion for this sentence:
Tarantool versions in each release series follow the same rules: